### PR TITLE
chore: updating the ISSUE_TEMPLATE for contributions.

### PR DESCRIPTION
**Pull request by Uyadav207**
This PR is regarding #24 issue, 

**Need**

- Necessary for an OSS project
- Helps maintainer as well contributor to have more specificity.

@pranshuchittora, please let me know if the PR is **Good to merge**.

Thanks :)

### CODE REVIEW

**Review by tewarig — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+<!--  A clear and concise description of what the bug is. -->
+
+**To Reproduce (Required)**
+
+> We highly recommend that you re-create the bug on [Snack](https://snack.expo.io). If not, list the steps that a reviewer can take to reproduce the behaviour:
+
+<!--
+Example:
+
+```bash
```
Comment (pranshuchittora) [inline .github/ISSUE_TEMPLATE/bug_report.md:31]: Why code block

Comment (Uyadav207) [reply .github/ISSUE_TEMPLATE/bug_report.md:31]: That's a typo... May i correct it

Comment (Uyadav207) [reply .github/ISSUE_TEMPLATE/bug_report.md:31]: @pranshuchittora , actually the steps.... need to be hidden, so only commenting the steps didn't worked.. I have wrapped the steps in a code block.

Comment (Uyadav207) [reply .github/ISSUE_TEMPLATE/bug_report.md:31]: @pranshuchittora , May I change or leave it as it is ?

Comment (pranshuchittora) [reply .github/ISSUE_TEMPLATE/bug_report.md:31]: Remove the code block and it should be outside the comment as
1.
2.
3.

Like this without any content
```
1.
2.
3.
```

Comment (Uyadav207) [reply .github/ISSUE_TEMPLATE/bug_report.md:31]: ok will do it asap

**Comment by Uyadav207** [comment]
Done with changes, @pranshuchittora

**Comment by pranshuchittora** [comment]
Just skimmed over the content.
LGTM :+1: 
Thanks

**Pull request merged**